app/assets/stylesheets/layouts/_tiles.scss

```diff
@@ -58,3 +57,16 @@
     }
   }
 }
+
+.l-tile__archive {
+  float: right !important;
```

Comment:
I wrangled with it for a while, there's a fair bit of nth-child stuff to do with the .l-tile which is overriding it - this was the easiest and quickest way out! - I'll take another look it tomorrow morning.